scripts/pin_the_slow_way.py

```diff
@@ -306,7 +306,12 @@ def create_update_pr(clone, remote_head, fork_remote, upstream_remote):
 
                 for pos, dep in enumerate(section):
                     for name, pin in pinned.items():
-                        if re.match(r"^\s*%s\s*$" % name, dep) and dep != pin:
+                        dep_split = dep.split(' ', 1)
+                        actual_name = dep_split[0]
+                        # skip star pins: https://github.com/conda-forge/conda-forge.github.io/issues/270
+                        if len(dep_split) > 1 and dep_split[1] == '*':
+                            continue
```

Comment:
It would IMHO be clearer to also handle this with a regex but I also don't immediately see any flaws in this logic.

Reply:
I had unittests in one of the stale PRs: https://github.com/conda-forge/conda-smithy/pull/482/files#diff-c81609e53a452e32dfd828bd02e72967.

Reply:
Ah nice - might be worthwhile to cherry pick those on top of this branch here or to a separate branch.